setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup
+
+import bus_sdk
+
+
+with open('requirements.txt') as f:
+    install_requires = f.read().splitlines()
+
+
+setup(
+    name='bus_sdk',
```

Comment:
Is it work have a vendor name in there? (`elife_bus_sdk`)

Reply:
I am happy to add vendor, I was just basing it on the previous suggestion. Will add.